### PR TITLE
skip token creation

### PR DESCRIPTION
**Pull request by jmontleon**

### CODE REVIEW

**Comment by jmontleon** [comment]
The downside of this is that I must create the labels path before creating the label. If any labels exist they are wiped out, but from what I see so far 1.) Tokens don't have labels, 2.) Since we aren't planning to (re)create them it shouldn't be a problem if labels are removed.

File: crane-export-playbook.yml [hunk 1]
```diff
@@ -28,5 +28,19 @@
...
+    find:
+      path: "{{ playbook_dir }}/transform"
+      recurse: true
+      patterns: 'transforms-Secret.*token.*yaml'
+      use_regex: true
+    register: tokens
+
+  - name: Add token label json patch
+    copy:
+      content: '[{"op":"add","path":"/metadata/labels", "value": {}},{"op":"add","path":"/metadata/labels/exported-by", "value":"crane2"}]'
```
Comment (shawn-hurley) [inline crane-export-playbook.yml:41]: Wonder if this removed labels that are important to folks?

**Review by shawn-hurley — APPROVED** [review]
If we do not see labels already ignore this and LGTM

if there are labels, maybe we need to assume there are labels?

**Comment by jmontleon** [comment]
Called this out in my comment above:
```

**Comment by jmontleon** [comment]
Mentioned this in my comment https://github.com/water-hole/crane-batch-migration/pull/1#issuecomment-843250852

The labels created by openshift don't seem to have labels. Since this is only touching the tokens we don't plan to recreate I don't see a problem. I feel like doing checks in ansible is going to get messy and slow. Definitely an area for improvement within crane as we get to it, but I think this might be somewhat challenging to overcome and maybe why it's not supported in json patch itself.

**Comment by shawn-hurley** [comment]
I agree you convinced me makes sense /lgtm /approve

**Pull request merged**